freud/order.pyx

```diff
@@ -249,25 +249,44 @@ cdef class Hexatic(_PairCompute):
     :math:`n` neighbors :math:`j` is given by:
 
     :math:`\psi_k \left( i \right) = \frac{1}{n}
-    \sum_j^n e^{k i \phi_{ij}}`
+    \sum_j^n e^{i k \phi_{ij}}`
```

Comment:
I swapped the order on purpose from `ki` to `ik` so that the `i` is more obviously an imaginary number in the context of `e^i`. I recognize this issue -- I tried to think of other options but `j, k` don't work because `k` is the hexatic order ... we could use `\alpha, \beta` to denote particles like some literature but I find that confusing sometimes as well. Possibly `m, n` if that dodges other uses of quantum numbers. I think I'm going to leave this as-is for now. I would recommend fixing this with a pull request that addresses this globally and is consistent across all modules.

Reply:
Yeah, most of the conventional letters are already used and if the `i,j` indices are consistently used throughout the documentation, then it could be confusing to switch in just one place.